nvm.sh

```diff
@@ -2357,6 +2357,9 @@ nvm() {
         setopt shwordsplit
       fi
       nvm_err "nvm --version: v$(nvm --version)"
+      if [ -n "$TERM_PROGRAM" ]; then
```

Comment:
the one inside the condition is safe; i'm pretty sure the one in the condition itself is not